### DISCUSSION TITLE
KAKUTE F7 in inav-configurator there is no mode NAV POSHOLD

### DISCUSSION THREAD

**Discussion by Vittamin** [iNavFlight/inav-configurator#1552] [Q&A] [answered]
Hello everyone
I'm a newbie. I have a quadcopter with a kakute f7 FC.
I want to turn on the flight mode NAV POSHOLD and RTH.
Downloaded installed 4.0.1., flashed the controller, but there are no these modes.
Tell me what can be done?

**Comment by DzikuVx** [comment, marked as answer]
You need both GPS and magnetometer up and runnig

**Comment by Vittamin** [comment]
Thank you very much!!!
When connecting the sensor and battery, the career poshold mode appeared!
You are a super person that helped a beginner)
